LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2019 Truffle Blockchain Group, Inc.
+Copyright (c) 2015-2023 Truffle Blockchain Group, Inc.
```

Comment:
```suggestion
Copyright (c) 2015-2023 ConsenSys Software Inc.
```

Reply:
I think you can make a separate PR for this, since I have no clue if this change is legal.

Reply:
TBG transferred all IP to ConsenSys Software Inc as part of the acquisition.

Reply:
> I have no clue if this change is legal.

It is. Oh well I figured you might want to address two lapses in the text in one PR.